### DISCUSSION TITLE
Quilt Compat

### DISCUSSION THREAD

**Discussion by RealSonarS** [shedaniel/RoughlyEnoughItems#884] [Q & A] [answered]
Right now trying to use REI with the Quilt Modloader causes this permanently.

![image](https://user-images.githubusercontent.com/80487755/171611721-2f0b070a-178f-4bc0-aed8-2d9f536978b7.png)

**Comment by shedaniel** [comment, marked as answer]
Do you have any logs in latest.log or rei.log?

**Reply by RealSonarS** [reply]
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/8826452/rei.log)


Apologies for the delay.

**Reply by RealSonarS** [reply]
[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/8826455/latest.log)

put both  in case.

**Reply by shedaniel** [reply]
Your issue is caused by another mod crashing during the recipe sync process, you will find that the vanilla recipe book doesn’t work properly. So this isn’t a quilt nor a REI issue.

The actual crash is this, which means that someone has messed up their recipes, and this is unfortunately not something REI would interfere to fix. Please identify the problematic mod yourselves.
[Render thread/ERROR]: Error executing task on Client
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1799.method_7909()" because the return value of "net.minecraft.class_1860.method_8110()" is null
